renovate.json

```diff
@@ -46,5 +46,33 @@
       ],
       "automerge": true
     }
+  ],
+  "regexManagers": [
+    {
+      "fileMatch": ["^docs/tutorials/ofo.md$"],
+      "matchStrings": ["open-feature\\/open-feature-operator\\/releases\\/download\\/(?<currentValue>.*?)\\/release\\.yaml"],
+      "depNameTemplate": "open-feature/open-feature-operator",
+      "datasourceTemplate": "github-releases"
+    },
+    {
+      "fileMatch": ["^docs/tutorials/getting-started/java.mdx$"],
+      "matchStringsStrategy": "any",
+      "matchStrings": [
+        "\\s*<groupId>dev\\.openfeature<\\/groupId>\\s*<artifactId>sdk<\\/artifactId>\\s*<version>(?<currentValue>.*?)<\\/version>",
+        "'dev\\.openfeature:sdk:(?<currentValue>.*?)'"
+      ],
+      "depNameTemplate": "dev.openfeature:sdk",
```

Comment:
The documentation for using `maven` as the `datasourceTemplate` is terrible, had to jump into [the source code](https://github.com/renovatebot/renovate/blob/2df8967ae27851e79d7c665483456825a21a6e6d/lib/modules/datasource/maven/util.ts#L284) to work out that a `:` was needed here. Perhaps this is obvious to people more familiar with maven